vendor/assets/javascripts/jquery.chapman_social_feed.js

```diff
@@ -245,6 +266,22 @@ ChapmanSocialFeed.prototype.realtimeSubscriptionChannel =  function() {
  */
 
 ChapmanSocialFeed.prototype.__realtimePostReceive = function(post) {
+  //console.log('ChapmanSocialFeed.__realtimePostReceive:', post);
+
+  // If feed has a keyword, make sure post contains that keyword.
+  // See Issue #196: https://github.com/chapmanu/signage/issues/196
+  if ( this.keyword ) {
+    // Checks for keyword in post (case-insensitive).
+    var insensitivePost = post.toLowerCase();
+    var insensitiveKeyword = this.keyword.toLowerCase();
+    var includesKeyword = insensitivePost.indexOf(insensitiveKeyword) !== -1;
+
+    if ( ! includesKeyword ) {
+      //console.log('Realtime post not displayed. Does not contain keyword', this.keyword);
```

Comment:
Is there a particular reason we are leaving console log statements in here?

Reply:
Yes, there's a comment with the first pair in the constructor that tries to cover this: it may be helpful for future troubleshooting. It's also meant to signal that we don't have a reliable way to really test this.

Reply:
It also explains what the `return` statement is doing (if it wasn't clear.)